_content/_docs_signals/context_checks.md

```diff
@@ -20,7 +20,7 @@ description:
 
 Each watch can define as many inputs, transformations, calculations and conditions as required, in any order.
 
-Each step in  the execution chain is called a `check` and needs to have a unique name. Example:
+Each step in  the execution chain is called a *check*. Example:
```

Comment:
Right now, names of checks are in fact not mandatory (this is in contrast to actions).
Reasons: a) There is no technical requirement for identification so far. b) It might be a bit too elaborate if I have to give every single condition a name.